hedera-node/yahcli/src/main/java/com/hedera/services/yahcli/output/FileYahcliOutput.java

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: Apache-2.0
+package com.hedera.services.yahcli.output;
+
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.services.bdd.spec.queries.QueryUtils;
+import com.hedera.services.yahcli.config.ConfigManager;
+import com.hedera.services.yahcli.suites.Utils;
+import com.hederahashgraph.api.proto.java.FileID;
+import com.hederahashgraph.api.proto.java.Response;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public final class FileYahcliOutput implements YahcliOutput, Closeable {
```

Comment:
If we're going to modify the output pattern, file and console loggers seem like a more robust option 🤔